helm-charts/common/ipexllm/values.yaml

```diff
@@ -0,0 +1,139 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for ipexllm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+accelDevice: a770
+
+# Configurations for OPEA microservice ipexllm
+LLM_MODEL_ID: deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B
```

Comment:
In the helm chart, i see the LLM_MODEL_ID is used for param '--model' and '--served-model-name'. 
The model has been downloaded by the initcontainer at `/data/models--{{ replace "/" "--" .Values.LLM_MODEL_ID }`. Can the ipexllm find the downloaded data only using the `$LLM_MODEL_ID`?
For param '--served-model-name', the helm chart tries to trim the prefix '/data/'. But the input here does not include the '/data/' prefix. Any special case?

Reply:
yes, it can. If LLM_MODEL_ID doesn't have /data/ prefix, the --served-model-name will be the same as LLM_MODEL_ID